README.md

```diff
@@ -13,3 +13,13 @@ Code Climate Issues | [![Issue Count](https://codeclimate.com/github/doge-life/d
 # doge-application
 The Front-end application of Doge Life.
 
+## Connecting to the proxy 
+
+If you are building a Docker image on the AscenaData network, you will need to connect to a proxy. The packer build is looking for the following environment variables to be set: 
+
+ASCENA_PROXY_PORT  
+ASCENA_PROXY_HOST  
+ASCENA_PROXY_UNAME  
+ASCENA_PROXY_PASSWORD
```

Comment:
It might be good to give an example.

```bash
export ASCENA_PROXY_UNAME='LTDTOO\your_username'
export ASCENA_PROXY_PASSWORD='your%20password'
export ASCENA_PROXY_HOST='l02piproxy01.corp.local'
export ASCENA_PROXY_PORT='80'
```